chrome/content_script.js

```diff
@@ -897,6 +899,7 @@ chrome.runtime.onMessage.addListener(
      * @param {string} msg JSON object containing valid password lengths.
      */
     function(msg) {
+      console.log("received message from service_worker: ", msg);
```

Comment:
does this message contain any information that shouldn't be logged to the console?